src/planner/plannodes/aggregate_plan_node.cpp

```diff
@@ -136,6 +136,20 @@ std::vector<std::unique_ptr<parser::AbstractExpression>> AggregatePlanNode::From
   return exprs;
 }
 
+bool AggregatePlanNode::RequiresCleanup() const {
+  return std::any_of(aggregate_terms_.begin(), aggregate_terms_.end(),
+                     [](auto agg_term) { return agg_term->RequiresCleanup(); });
+}
```

Comment:
Nit of all nits: add a blank line between these two functions.

Reply:
Done